### PR TITLE
Show import from CMS button for all section types in admin

### PR DESCRIPTION
**Pull request by coderdan**

### CODE REVIEW

**Comment by freshtonic** [comment]
LGTM

File: app/dashboards/agency_dashboard.rb [hunk 1]
```diff
@@ -2,19 +2,7 @@
 
 class AgencyDashboard < SectionDashboard
 
-  ATTRIBUTE_TYPES = {
-      # The import govcms button needs to know the CMS type and the object id
-      # We use #itself here so the views can see the entire object
```
Comment (tessereth) [inline app/dashboards/agency_dashboard.rb]: It might be useful to move this comment to sections dashboard as well

**Comment by maxious** [comment]
LGTM

**Comment by tessereth** [comment]
Other than one minor comment, LGTM

**Pull request merged**